src/main/java/com/solvd/shop/services/shop/StatusService.java

```diff
@@ -0,0 +1,57 @@
+package com.solvd.shop.services.shop;
+
+import com.solvd.shop.interafaces.shop.IStatusDAO;
+import com.solvd.shop.jdbc.dao.shop.StatusDAO;
+import com.solvd.shop.models.shop.Status;
+import com.solvd.shop.util.ConnectionPool;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public class StatusService implements IStatusDAO<Status> {
+
+    private static ConnectionPool connectionPool;
+    private static final Logger LOGGER = LogManager.getLogger(StatusService.class);
+
+    private static final StatusDAO statusDAO = new StatusDAO(connectionPool);
+
+    static {
+        try {
+            connectionPool = ConnectionPool.getInstance();
+        } catch (SQLException e) {
+            LOGGER.info(e);
+        }
+    }
+
+    @Override
```

Comment:
when previous comment is resolved you can remove these static blocks from your classes
```suggestion
    private static ConnectionPool connectionPool = ConnectionPool.getInstance();
    private static final StatusDAO statusDAO = new StatusDAO(connectionPool);
    private static final Logger LOGGER = LogManager.getLogger(StatusService.class);
    
    @Override
```